plugins/console/index.js

```diff
@@ -0,0 +1,102 @@
+var q = require('q');
```

Comment:
It's super confusing, but Protractor actually uses both `q` and the webdriver promises. I think it's OK to use `q` in this situation, because it has no need to sync up with the webdriver promises - and it's actually probably safer, because it can't accidentally interfere with the webdriver control flow.